src/editor.c

```diff
@@ -671,9 +672,13 @@ static gboolean reshow_calltip(gpointer data)
 
 	g_return_val_if_fail(calltip.sci != NULL, FALSE);
 
-	SSM(calltip.sci, SCI_CALLTIPCANCEL, 0, 0);
 	doc = document_get_current();
 
+	if (plugin_extension_calltips_provided(doc))
+		return FALSE;
```

Comment:
For this patch yes.

I think the placement was some left-over of my attempt to rewrite the reshowing code (because the current one doesn't work very well), but it should be done separately.

Reply:
Looking forward for the additional fixes :)